helper-contract/src/helper_contract.rs

```diff
@@ -0,0 +1,51 @@
+#![no_std]
+
+#[allow(unused_imports)]
+use multiversx_sc::imports::*;
+
+use transaction::{EthTransaction, EthTxAsMultiValue};
+use sc_proxies::bridge_proxy_contract_proxy;
+
+#[multiversx_sc::contract]
+pub trait HelperContract {
+    #[init]
+    fn init(&self) {}
+
+    #[upgrade]
+    fn upgrade(&self) {}
+
+    #[view(getCalleeAddress)]
```

Comment:
Storage mapper usually go last.
Start with init, upgrade, endpoints and end with storages.